benchexec/tools/veriabs.py

```diff
@@ -0,0 +1,73 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import xml.etree.ElementTree as ET
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+	veriabs for SV-COMP 2017
+    """
+    def executable(self):
+        return util.find_executable('veriabs')
+
+
+    def version(self, executable):
+        return " Version 0.001 Alpha"
+
+
+    def name(self):
+        return 'VeriAbs SV-COMP 2017'
```

Comment:
The tool-info module in BenchExec should not be specific to SV-Comp and particularly not to a specific year of SV-Comp. This module should be usable for all people that want to run benchmarks with BenchExec and your tool. So please change the name to something like `VeriAbs`.